### ISSUE TITLE
Can we reduce the changes to zkEVM etc?

### ISSUE THREAD

**Issue by edmundedgar** [RealityETH/subjectivocracy#270]
May be related to #269 

We made changes to zkEVM to make it forkable which then have to be audited. For better compatibility and to reduce the amount that has to be audited, it may be possible to run the stock zkEVM, although we probably still need to tinker with the bridges.

Something like:

 * The CreateChildren functionality is moved out of the zkEVM contract into a manager contract
 * After a fork, the new chains are created as... new chains, with the starting genesis hash from the old chain
 * Since we may not be able to make the change we made to the batch number etc we may have to tinker with the sequencer, rpc node etc to make the infrastructure work

**Comment by josojo** [comment]
Yeah, I checked when we used the `onlyBeforeForking` modifier and it seems to work as always the emergency is also checked.

It's nice that we reduce the diff... but I am not sure how much we are really winning by that. Since the interplay with the bridge needs to be audited anyways and generally the code would become harder to read.

I mean its not unreasonable to do, but it also not plain obvious that its better
